libs/dh/metering-point/domain/src/lib/is-metering-point-id.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { meteringPointIdPattern } from './metering-point-id-pattern';
+
+/**
+ * Validates that a metering point id is exactly 18 digits
```

Comment:
Consider removing the amount of digits from the comment, otherwise it has to be maintained if the regex changes